moped-toolbox/component_geo_processing/settings.py

```diff
@@ -6,7 +6,6 @@
 PROJECT_LIST_KEYS = [
     "project_id",
     "project_description",
-    "completion_end_date",
```

Comment:
thanks for cleaning up this tool too! 🧹 I don't see a trace of this column anywhere except for old migrations. 🙏